core/src/sumeragi/message.rs

```diff
@@ -76,6 +76,22 @@ pub enum Message {
     ViewChangeSuggested,
 }
```

Comment:
```suggestion
}
```

IMO this shouldn't be mixed with the data message anymore